sdk/search/azure-search/src/test/java/com/azure/search/SearchServiceTestBase.java

```diff
@@ -113,11 +113,12 @@
     @BeforeAll
     public static void beforeAll() {
         initializeAzureResources();
+        azureSearchResources.createResourceGroup();
     }
 
     @AfterAll
     public static void afterAll() {
-        azureSearchResources.deleteResourceGroup();
+        //azureSearchResources.deleteResourceGroup();
```

Comment:
Any particular reason we are adding a call the resource group creation but removing one to group deletion?

Reply:
One resource group for the entire tests, so there is no need to delete for every test class